specs/latest/2.0/index.html

```diff
@@ -2882,6 +2883,39 @@ <h2 class="no-toc">Table of contents</h2>
         The error condition is added to ensure consistent behavior across all platforms.
     </div>
 
+    <h3>Color values from a fragment shader must match the color buffer format</h3>
+
+    <p>
+        Color values written by a fragment shader may be floating-point, signed integer, or unsigned
+        integer. If the values written by the fragment shader do not match the format(s) of the
+        corresponding color buffer(s), the result is undefined in OpenGL ES 3.0  <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-3.9.2">OpenGL ES 3.0.4 &sect;3.9.2</a>)</span>. In WebGL, generates
+        an <code>INVALID_OPERATION</code> error in the corresponding draw call, including
+        <code>drawArrays</code>, <code>drawElements</code>, <code>drawArraysInstanced
+        </code>, <code>drawElementsInstanced </code>, and <code>drawRangeElements</code>.
+    </p>
+    <p>
+        If the color buffer has a normalized fixed-point format, floating-point color
+        values are converted to match the format; generates no error in such situation.
+    </p>
+
+    <h3>A sampler type must match the internal texture format</h3>
+
+    <p>
+        Texture lookup functions return values as floating point, unsigned integer or signed integer,
+        depending on the sampler type passed to the lookup function. If the wrong sampler type is
+        used for texture access, i.e., the sampler type does not match the texture internal format,
+        the returned values are undefined in OpenGL ES Shading Language 3.00.4
+        <span class="gl-spec">(<a href="https://www.khronos.org/registry/gles/specs/3.0/GLSL_ES_Specification_3.00.4.pdf#nameddest=section-8.8">OpenGL ES Shading Language 3.00.4 &sect;8.8</a>)</span>.
+        In WebGL, generates an <code>INVALID_OPERATION</code> error in the corresponding draw call,
+        including <code>drawArrays</code>, <code>drawElements</code>, <code>drawArraysInstanced</code>,
+        <code>drawElementsInstanced </code>, and <code>drawRangeElements</code>.
```

Comment:
Generating INVALID_OPERATION seems fine -- failing fast will probably be more obvious to the developer. BTW, I went back through the audio of the last conference call and there was no resolution for this particular topic.